app/components/InnovationsSection/InnovationsSection.tsx

```diff
@@ -0,0 +1,47 @@
+import { InnovationCard } from "./InnovationCard";
+
+export const InnovationsSection = () => {
+  const innovations = [
```

Comment:
This array is not changeable, please put it outside of component, can be the same file